tests/smoke.runner.js

```diff
@@ -5,29 +5,24 @@ import assert from 'assert'
 import launch from './helpers/launch'
 import { SERVICE_LOGS, LAUNCHER_LOGS, REPORTER_LOGS } from './helpers/fixtures'
 
-const sleep = (ms) => new Promise(resolve => setTimeout(resolve, ms));
+const sleep = (ms) => new Promise(resolve => setTimeout(resolve, ms))
 
-(async () => {
-    /**
-     * Mocha wdio testrunner tests
-     */
```

Comment:
I don't think these comments add a lot of time to the execution, I rather would keep them

Reply:
yeap, will revert this

Reply:
@christian-bromann done.